### PR TITLE
QA-289: Review and publish testing tool and docs

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
Best reviewed commit by commit :wink:

**Review by tranchitella — APPROVED** [review]

**Comment by tranchitella** [comment]
@0lmi I'll wait for your review before merging the PR

**Pull request merged**
